packs/data/pfs-season-3-bestiary.db/filth-fire-pfs-3-01.json

```diff
@@ -396,7 +396,7 @@
             "type": "lore"
         }
     ],
-    "name": "Filth Fire (3-01)",
+    "name": "Filth Fire (PFS 3-01)",
```

Comment:
Does not appear to be different from the existing Bestiary 2 Filth Fire

Reply:
Alignment is N, not NE